### PR TITLE
feat(route): add 集思录广场

### LINKED ISSUE #17962: 集思录30天热门有问题
### 路由地址

```routes
NOROUTE
```

### 完整路由地址

```fullroutes
rsshub://jisilu/hot/30
```

### 相关文档

https://docs.rsshub.app/routes/bbs#%E9%9B%86%E6%80%9D%E5%BD%95

### 预期是什么？

30天内的热门帖子

### 实际发生了什么？

实际没人回复的帖子都能算作30天热门
PixPin_2024-12-23_13-45-48

### 部署

RSSHub 演示 (https://rsshub.app)

### 部署相关信息

_No response_

### 额外信息

```shell
经常发生 而且更新频率略频繁 30天内的热贴应该是很少的
```

### 这不是重复的 issue

- [X] 我已经搜索了 [现有 issue](https://github.com/DIYgod/RSSHub/issues)，以确保该错误尚未被报告。

### PR DESCRIPTION
**Pull request by nczitzk**
## Involved Issue / 该 PR 相关 Issue

Close #17962

## Example for the Proposed Route(s) / 路由地址示例

```routes
/jisilu/explore/category-4__sort_type-hot__day-30
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [x] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [x] Parsed / 可以解析
  - [x] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/routes/jisilu/category.ts [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+import { rootUrl, processItems } from './util';
+import InvalidParameterError from '@/errors/types/invalid-parameter';
+
+export const handler = async (ctx: Context): Promise<Data> => {
+    const { id } = ctx.req.param();
+
+    if (!id) {
+        throw new InvalidParameterError('请填入合法的分类 id，参见广场 https://www.jisilu.cn/explore/');
+    }
```
Comment (TonyRL) [inline lib/routes/jisilu/category.ts:16]: Unreachable condition. https://github.com/DIYgod/RSSHub/blob/87df3dc03815e7e3a2f43ec3c5acb5378298594e/lib/routes/jisilu/category.ts#L51 defines `id` as compulsory parameter. User who visiting the route without providing `id` will see a `NotFoundError` instead

File: lib/routes/jisilu/category.ts [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        },
+        {
+            title: '基金',
+            source: ['www.jisilu.cn/category/7'],
+            target: '/jisilu/category/7',
+        },
+        {
+            title: '股票',
+            source: ['www.jisilu.cn/category/8'],
+            target: '/jisilu/category/8',
```
Comment (TonyRL) [inline lib/routes/jisilu/category.ts:115]:
```suggestion
            target: '/category/3',
        },
        {
            title: '债券/可转债',
            source: ['www.jisilu.cn/category/4'],
            target: '/category/4',
        },
        {
            title: '套利',
            source: ['www.jisilu.cn/category/5'],
            target: '/category/5',
        },
        {
            title: '其他',
            source: ['www.jisilu.cn/category/6'],
            target: '/category/6',
        },
        {
            title: '基金',
            source: ['www.jisilu.cn/category/7'],
            target: '/category/7',
        },
        {
            title: '股票',
            source: ['www.jisilu.cn/category/8'],
            target: '/category/8',
```

File: lib/routes/jisilu/people.ts [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    questions: '101',
+    answers: '201',
+};
+
+export const handler = async (ctx: Context): Promise<Data> => {
+    const { id, type = 'questions' } = ctx.req.param();
+
+    if (!id) {
+        throw new InvalidParameterError('请填入合法的用户 id，参见用户排名 https://www.jisilu.cn/users/');
+    }
```
Comment (TonyRL) [inline lib/routes/jisilu/people.ts:21]: Unreachable condition. https://github.com/DIYgod/RSSHub/blob/87df3dc03815e7e3a2f43ec3c5acb5378298594e/lib/routes/jisilu/people.ts#L64 defines `id` as compulsory parameter. User who visiting the route without providing `id` will see a `NotFoundError` instead

File: lib/routes/jisilu/topic.ts [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+import { rootUrl, processItems } from './util';
+import InvalidParameterError from '@/errors/types/invalid-parameter';
+
+export const handler = async (ctx: Context): Promise<Data> => {
+    const { id } = ctx.req.param();
+
+    if (!id) {
+        throw new InvalidParameterError('请填入合法的话题 id，参见话题广场 https://www.jisilu.cn/topic');
+    }
```
Comment (TonyRL) [inline lib/routes/jisilu/topic.ts:16]: Unreachable condition. https://github.com/DIYgod/RSSHub/blob/87df3dc03815e7e3a2f43ec3c5acb5378298594e/lib/routes/jisilu/topic.ts#L47 defines `id` as compulsory parameter. User who visiting the route without providing `id` will see a `NotFoundError` instead

File: lib/routes/jisilu/people.ts [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['www.jisilu.cn/people/:id'],
+            target: (params) => {
+                const id = params.id;
+
+                return `/jisilu/people/${id}`;
+            },
```
Comment (TonyRL) [inline lib/routes/jisilu/people.ts:99]:
```suggestion
            target: '/people/:id',
```

File: lib/routes/jisilu/topic.ts [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['www.jisilu.cn/topic/:id'],
+            target: (params) => {
+                const id = params.id;
+
+                return `/jisilu/topic/${id}`;
+            },
```
Comment (TonyRL) [inline lib/routes/jisilu/topic.ts:81]:
```suggestion
            target: '/topic/:id',
```

File: lib/routes/jisilu/category.ts (hunk 1 above)
Comment (nczitzk) [reply lib/routes/jisilu/category.ts:16]: Thanks for the reviews. 😊

**Pull request merged**